meson.build

```diff
@@ -0,0 +1,845 @@
+'Eigen/src/QR/ColPivHouseholderQR.h',
+'Eigen/src/QR/ColPivHouseholderQR_LAPACKE.h',
+'Eigen/src/QR/CompleteOrthogonalDecomposition.h',
+'Eigen/src/QR/FullPivHouseholderQR.h',
+'Eigen/src/QR/HouseholderQR.h',
+'Eigen/src/QR/HouseholderQR_LAPACKE.h',
+'Eigen/src/SparseCholesky/SimplicialCholesky.h',
+'Eigen/src/SparseCholesky/SimplicialCholesky_impl.h',
+'Eigen/src/SparseCore/AmbiVector.h',
+'Eigen/src/SparseCore/CompressedStorage.h',
+'Eigen/src/SparseCore/ConservativeSparseSparseProduct.h',
+'Eigen/src/SparseCore/MappedSparseMatrix.h',
+'Eigen/src/SparseCore/SparseAssign.h',
+'Eigen/src/SparseCore/SparseBlock.h',
+'Eigen/src/SparseCore/SparseColEtree.h',
+'Eigen/src/SparseCore/SparseCompressedBase.h',
+'Eigen/src/SparseCore/SparseCwiseBinaryOp.h',
+'Eigen/src/SparseCore/SparseCwiseUnaryOp.h',
+'Eigen/src/SparseCore/SparseDenseProduct.h',
+'Eigen/src/SparseCore/SparseDiagonalProduct.h',
+'Eigen/src/SparseCore/SparseDot.h',
+'Eigen/src/SparseCore/SparseFuzzy.h',
+'Eigen/src/SparseCore/SparseMap.h',
+'Eigen/src/SparseCore/SparseMatrix.h',
+'Eigen/src/SparseCore/SparseMatrixBase.h',
+'Eigen/src/SparseCore/SparsePermutation.h',
+'Eigen/src/SparseCore/SparseProduct.h',
+'Eigen/src/SparseCore/SparseRedux.h',
+'Eigen/src/SparseCore/SparseRef.h',
+'Eigen/src/SparseCore/SparseSelfAdjointView.h',
+'Eigen/src/SparseCore/SparseSolverBase.h',
+'Eigen/src/SparseCore/SparseSparseProductWithPruning.h',
+'Eigen/src/SparseCore/SparseTranspose.h',
+'Eigen/src/SparseCore/SparseTriangularView.h',
+'Eigen/src/SparseCore/SparseUtil.h',
+'Eigen/src/SparseCore/SparseVector.h',
+'Eigen/src/SparseCore/SparseView.h',
+'Eigen/src/SparseCore/TriangularSolver.h',
+'Eigen/src/SparseLU/SparseLU.h',
+'Eigen/src/SparseLU/SparseLUImpl.h',
+'Eigen/src/SparseLU/SparseLU_column_bmod.h',
+'Eigen/src/SparseLU/SparseLU_column_dfs.h',
+'Eigen/src/SparseLU/SparseLU_copy_to_ucol.h',
+'Eigen/src/SparseLU/SparseLU_gemm_kernel.h',
+'Eigen/src/SparseLU/SparseLU_heap_relax_snode.h',
+'Eigen/src/SparseLU/SparseLU_kernel_bmod.h',
+'Eigen/src/SparseLU/SparseLU_Memory.h',
+'Eigen/src/SparseLU/SparseLU_panel_bmod.h',
+'Eigen/src/SparseLU/SparseLU_panel_dfs.h',
+'Eigen/src/SparseLU/SparseLU_pivotL.h',
+'Eigen/src/SparseLU/SparseLU_pruneL.h',
+'Eigen/src/SparseLU/SparseLU_relax_snode.h',
+'Eigen/src/SparseLU/SparseLU_Structs.h',
+'Eigen/src/SparseLU/SparseLU_SupernodalMatrix.h',
+'Eigen/src/SparseLU/SparseLU_Utils.h',
+'Eigen/src/SparseQR/SparseQR.h',
+'Eigen/src/SPQRSupport/SuiteSparseQRSupport.h',
+'Eigen/src/StlSupport/details.h',
+'Eigen/src/StlSupport/StdDeque.h',
+'Eigen/src/StlSupport/StdList.h',
+'Eigen/src/StlSupport/StdVector.h',
+'Eigen/src/SuperLUSupport/SuperLUSupport.h',
+'Eigen/src/SVD/BDCSVD.h',
+'Eigen/src/SVD/JacobiSVD.h',
+'Eigen/src/SVD/JacobiSVD_LAPACKE.h',
+'Eigen/src/SVD/SVDBase.h',
+'Eigen/src/SVD/UpperBidiagonalization.h',
+'Eigen/src/UmfPackSupport/UmfPackSupport.h'
+)
+
+install_headers(eigen_headers, subdir: 'eigen3')
+
+eigens_dep = declare_dependency(
```

Comment:
There is a rogue `s` in the middle :(
`eigen_dep`

Reply:
GAGH butterfingers.